Dockerfile-grocy

```diff
@@ -38,11 +38,9 @@ RUN     apk update && \
         docker-php-ext-install -j${NPROC} gd
 
 # Extract application release package
-RUN     wget --header "Authorization: ${GITHUB_API_TOKEN}" -t 3 -T 30 -nv "https://github.com/grocy/grocy/releases/latest/download/grocy.tar.gz" && \
-        tar xzf grocy.tar.gz && \
-        rm grocy.tar.gz && \
-        mv grocy/* . && \
-        rm -r grocy-* && \
+RUN     wget --header "Authorization: ${GITHUB_API_TOKEN}" -t 3 -T 30 -nv -O "grocy.zip" "https://releases.grocy.info/latest" && \
```

Comment:
:warning: This is/was a potential API key leak for anyone running from this commit :/

Reply:
cc @berrnd ; I've pushed to remove the offending commit, but there was briefly a change here which could have had people sending GitHub API tokens your way.  I'm not sure there's a ton we can safely do about that, and I think it's unlikely that anyone experienced this, but just a FYI.  I may delete this thread shortly to lessen the chance this code path & broken build script would accidentally appear again in future.